crates/swc_common/src/syntax_pos.rs

```diff
@@ -423,6 +434,20 @@ impl Span {
             }
         }
     }
+
+    /// Dummy span, both position are extremely large numbers so they would be
+    /// ignore by sourcemap, but can still have comments
+    pub fn dummy_with_cmt() -> Self {
+        GLOBALS.with(|globals| {
+            globals.dummy_cnt.fetch_add(1, Ordering::SeqCst);
```

Comment:
Can't we use the return value of this?
We can add `1` after fetching it.

Reply:
Seems like you missed this?

Reply:
I don't know much about atomics, but if two threads fetch_add at same time, wouldn't they get same return value?

Reply:
No, fetch-add is a single atomic instruction and if `atomic` here means all operations of multiple threads are synchornized, and the unit is `fetch + add`.

So other thread cannot perform anything about this data (in memory) between `fetch` and `add`